config/locales/pt.yml

```diff
@@ -0,0 +1,10 @@
+pt:
+  activerecord:
+    attributes:
+      user:
+        name: "Nome"
+        submit: "Cenas"
```

Comment:
Huh?

Reply:
Sorry. I was testing things out and didn't noticed that I left this here.